do-catalog/src/store/actions/catalog.js

```diff
@@ -25,11 +25,11 @@ function filtersToPayload(filter) {
   payload += searchText.length ? `&searchtext=${searchText}` : '';
 
   //TODO: Make this filters dynamic too
-  payload += licenses.length ? `&license=${licenses.join(',')}` : '';
-  payload += categories.length ? `&category=${categories.join(',')}` : '';
-  payload += countries.length ? `&country=${countries.join(',')}` : '';
-  payload += geographies.length ? `&geography=${geographies.join(',')}` : '';
-  payload += sources.length ? `&provider=${sources.join(',')}` : '';
+  payload += licenses.length ? `&license=${licenses.join('&license=')}` : '';
```

Comment:
Just a suggestion, I think you can simplify by doing:

```
payload += licenses.map(v => `&license=${v}`).join('')
```

Reply:
Yes, I'm not sure if it worth the change. I think that right now it is easy enough to read and I personally won't add a second loop for each filter.

Reply:
For me, using map is cleaner, more readable and less verbose, there is no penalty performance too. But this is just a suggestion so :smile: